packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -977,6 +977,20 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 					task.title = `Default master language set to ${chalk.cyan(
 						"en-US",
 					)} 🇺🇸. You can change it anytime in your project settings.`;
+
+					try {
```

Comment:
⚠️ This code is at the wrong place, right? You're really focusing on your flow but for starter we also want to set that this as been done right?

Best place would be with the other `completeOnboardingStep` call of this file, no? So you directly set all steps as done directly together.

Reply:
isn't [that change](https://github.com/prismicio/slice-machine/pull/1487/commits/d2cc45f76ef85eb887d233e9382921c51570d079) making this change not needed anymore?
now we will set the language for every repo created with CLI, right?
Then in my opinion it's better to complete the step when it's actually being complete, so we don't complete it if `setDefaultMasterLocale` fail